src/table/utils/handle-scroll.ts

```diff
@@ -35,18 +35,18 @@ export const handleNavigateTop = (cellCoord, rootElement) => {
   if (x < MIN_ROW_COUNT) {
     tableContainer.scrollTo({
       top: 0,
-      behavior: 'instant',
+      behavior: 'auto',
```

Comment:
how does the behavior differ here?

Reply:
no difference. 'instant' is just not available in typescript
<img width="566" alt="Screenshot 2022-08-12 at 11 11 30" src="https://user-images.githubusercontent.com/4471489/184323439-f1b51f68-faff-401f-be9c-729e89a6704b.png">

Reply:
ok